tests/Tests/400__ods_dashboard/adminUITest.robot

```diff
@@ -0,0 +1,159 @@
+*** Settings ***
```

Comment:
file name should follow the usual naming convention  e.g. `413__ods_dashboard_user_mgmt`

Reply:
will do the changes